app/assets/stylesheets/about.css.scss

```diff
@@ -4,7 +4,7 @@
     .about-image {
         z-index: -20;
         background-size: cover;
-        background-image: url("s3-us-west-2.amazonaws.com/foodshift/assets/images/landing_splash_bridge.jpg");
+        background-image: url("http://s3-us-west-2.amazonaws.com/foodshift/assets/images/landing_splash_bridge.jpg");
```

Comment:
URL `http://s3-us-west-2.amazonaws.com/foodshift/assets/images/landing_splash_bridge.jpg` should not contain protocol or domain